bokeh/sphinxext/bokeh_plot.py

```diff
@@ -160,7 +161,7 @@ def run(self):
         js_name = f"bokeh-plot-{uuid4().hex}-external-{docname}.js"
 
         try:
-            (script, js, js_path, source) = _process_script(source, path, env, js_name)
+            (script, js, js_path, source, doc) = _process_script(source, path, env, js_name)
         except Exception as e:
             raise RuntimeError(f"Sphinx bokeh-plot exception: \n\n{e}\n\n Failed on:\n\n {source}")
         env.bokeh_plot_files[js_name] = (script, js, js_path, source, dirname(env.docname))
```

Comment:
The data in `env.bokeh_plot_files` is used later in the build process in the `build_finished` function. AFAICT  I think `script`, `js`, and `source` values are not actually used/needed there. Can you see if they can be removed from this tuple here (and in `build_finished`)? This might significantly reduce memory usage during the build.